src/view/ChewingEditor.cpp

```diff
@@ -128,6 +129,63 @@ void ChewingEditor::exportUserphrase(const QString& file)
     emit model_->exportUserphrase(exporter);
 }
 
+void ChewingEditor::showAbout()
+{
+/* Make about dialog shows like aboutQt() on Mac OS X.
+   So we have to prevent call twice on Mac OS X. */
+#ifdef Q_OS_MAC
+    static QPointer<QMessageBox> oldAboutBox;
+
+    if (oldAboutBox) {
+        oldAboutBox->show();
+        oldAboutBox->raise();
+        oldAboutBox->activeWindow();
+        return;
+    }
+#endif
+
+    QString translatedCaption = tr("%1%2")
+        .arg(tr("<h3>About Chewing Editor</h3>"))
+        .arg(tr("<p>Version %1 (%2).</p>")
+            .arg(PROJECT_VERSION)
+            .arg(PROJECT_GIT_VERSION));
+    QString translatedText = tr(
+        "<p>Chewing editor is a cross platform Chewing user phrase editor. "
+        "It provides a easy way to manage user phrase. With it, user can "
+        "customize their user phrase to increase input performance.</p>"
+
+        "<p>Chewing editor licensed under the GNU GPL version 2.1 or later version.</p>"
+
+        "<p>Please see <a href=\"https://github.com/chewing/chewing-editor/blob/master/COPYING\">COPYING</a> "
+        "for detail of Chewing editor licensing.</p>"
+
+        "<p>Copyright&copy; 2004–2016 by Chewing Core Team and other contributors.</p>"
+
+        "<p>Chewing editor is developed as an open source project on "
+        "<a href=\"https://github.com/chewing/chewing-editor\">github.com/chewing/chewing-editor</a>.</p>"
```

Comment:
I just copied the format from about Qt as screenshot below. If you think showing full url is better, I'll change this.

![image](https://cloud.githubusercontent.com/assets/841969/13994451/cd14f248-f15f-11e5-9237-bdbe8289789d.png)